docs/src/components/HeaderWithLink.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "styled-components";
+import { ButtonLink, Heading, Stack } from "@kiwicom/orbit-components";
+import { Link as LinkIcon } from "@kiwicom/orbit-components/icons";
+
+import { sluggify } from "../utils/common";
+
+const StyledAnchorWrapper = styled.div`
+  margin-top: -10px;
+
+  svg {
+    visibility: hidden;
+  }
+
+  &:hover svg {
+    visibility: visible;
```

Comment:
i would probably solve it via `opacity` and also add some transition, so the link will appear more smoothly :)